app/main.py

```diff
@@ -1 +1,54 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name, skills=None):
+        self.name = name
+        if skills is None:
+            self.skills = []
+
+    def learn_skill(self, skill):
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    def __init__(self, name):
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
```suggestion
        self.skills += ["JavaScript", "HTML", "CSS"]
```
Add the list of the required skills to the attribute. In that case, you shouldn't create a constructor for FullStackDeveloper class